integration/client/daemon.go

```diff
@@ -79,6 +82,25 @@ func (d *daemon) waitForStart(ctx context.Context) (*Client, error) {
 				}
 				continue
 			}
+			resp, perr := client.IntrospectionService().Plugins(ctx)
+			if perr != nil {
+				return nil, fmt.Errorf("failed to get plugin list: %w", perr)
+			}
+			var loadErr error
+			for _, p := range resp.Plugins {
+				if p.InitErr != nil && !strings.Contains(p.InitErr.Message, plugin.ErrSkipPlugin.Error()) {
+					pluginErr := fmt.Errorf("failed to load %s.%s: %s", p.Type, p.ID, p.InitErr.Message)
+					if loadErr == nil {
```

Comment:
There is no need to keep this `nil` check.
`errors.Join` already does that.

https://go.dev/play/p/EkcC-IUe3sV